storage/objects/acl.go

```diff
@@ -1,162 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
If we delete this file at the same time as adding the new samples, the existing docs that depend on this file will not be able to be published. So, we should either
* "freeze" the versions in the docs to a particular commit, or 
* keep this file, submit the PR, then update the docs, then delete this file.